src/components/membersItem.js

```diff
@@ -0,0 +1,59 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+const MembersItem = ({ name, image, location, techs, hobbies, links }) => (
+  <div>
+    {image && <img src={image} alt={name + `'s beautiful face`} style={{ maxWidth: 200 }}
+      onError={(event) => { event.target.onerror = null; event.target.src = "https://avatars3.githubusercontent.com/u/13358086?s=200&v=4" }}
```

Comment:
@galiotoweb Can you pull this out to a function similar to how it was before? It will help with readability